spec/spec_helper.rb

```diff
@@ -30,8 +30,16 @@ def load_response(filename, hash: false)
   response
 end
 
-def load_request(filename)
-  File.read(File.join(MOCK_DIR, 'requests', filename))
+def load_request(filename, hash: false)
```

Comment:
I know that you love code duplication but can you maybe merge this and `load_response` method? 😄